<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitTestData</class>
 <widget class="QMainWindow" name="UnitTestData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>877</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UnitTestData</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color : white;
background-color : rgb(49,48,43);

QLineEdit{
color: white;
background-color : rgb(38, 38, 36);
border-color: rgb(76, 76, 76);
}

QPushButton:disabled
{
  background-color: rgb(225,67,19) ;
  border: 2px solid #8f8f91;  
  border-radius: 6px;
 color: white ;
}

#InfoTab {
color: white;
background-color : rgb(49, 48, 43);
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="outputFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>921</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="outputFile">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>921</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonDir">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>openDirectory</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
     <item>
      <widget class="QListView" name="listView"/>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="inputFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>921</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="filterEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;FILTER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFilter">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="currentFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>921</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
